pkg/connector/handlematrix.go

```
@@ -239,3 +240,12 @@ func (tc *TwitterClient) HandleMatrixViewingChat(ctx context.Context, chat *brid
 	tc.client.SetActiveConversation(conversationID)
 	return nil
 }
+
+func (tc *TwitterClient) HandleMatrixDeleteChat(ctx context.Context, chat *bridgev2.MatrixDeleteChat) error {
```

This should also throw an error if DeleteForEveryone is set